packages/sdk-metrics/src/Instruments.ts

```diff
@@ -22,6 +22,7 @@ import {
   ValueType,
   UpDownCounter,
   Counter,
+  Gauge,
```

Comment:
This is the case I was talking about above, where we may need to duplicate the type from `@opentelemetry/api` as we need to be backward compatible to `@opentelemetry/api@1.3.0` which does not yet have this type. :slightly_smiling_face: